src/index.html

```diff
@@ -15,8 +15,45 @@
       rel="stylesheet"
       href="./style.css"
     />
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@100;500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <div class="header_top">
+        <div class="header_logo">
+          <img src="images/logo.png" alt="MOYO-logo">
```

Comment:
The `alt` attribute should describe the image content. The current `alt` attribute 'MOYO-logo' could be more descriptive about the content of the image.